### ISSUE TITLE
CSS Property borderColor is not applied to both borderLeft and borderRight

### ISSUE THREAD

**Issue by Jdyn** [cssinjs/jss#1161]
__Expected behavior:__
CSS Property `borderColor` shoud apply to both CSS Properties `borderLeft` and `borderRight`.

__Describe the bug:__
CSS Property `borderColor` is not being applied to both `borderLeft` and `borderRight`. It is only applied to `borderLeft`.

__Codesandbox link:__ 
[Sandbox](https://codesandbox.io/embed/react-jss-playground-s7lbe)

__Versions (please complete the following information):__
 - jss: 10.0.0-alpha.22
 - Browser [e.g. chrome, safari]: Chrome
 - OS [e.g. Windows, macOS]: Windows 10

**Comment by HenriBeck** [comment]
This is an issue with our sorting that we do. The longer a css property is, the more specific the property is in our code.

Now that `borderRight` and `borderColor` have the same length so we do normal string sorting. In this case, the `borderColor` needs to come after `borderRight` to override the proper value.

cc @kof 

You can easily fix this by moving your color into `borderRight`.

**Comment by Jdyn** [comment]
I see, It must have been a recent change in the plug-in because the issue doesn't occur in 8.6.1. Thank you for taking a look.

**Comment by HenriBeck** [comment]
Yes, this was changed due to different iterations order in node versions.

#1084
